src/window/User Guide.md

```diff
@@ -0,0 +1,132 @@
+% User Guide to Using Piston Windows
+
+# Guide to using Piston Windows
+
+## Getting Help
+
+*Notice! Piston is being actively developed. Please let us know if anything in this tutorial is outdated.*
+
+See the main [Piston Guide](../../GUIDE.md) for general help with Piston.
+
+## Overview
+
+The piston engine is designed to be an abstraction. It provides the same functionality through several different backends, each of which have their own quirks and extra features.
```

Comment:
Is this better?:

"The Piston engine is designed to be an abstraction that makes it possible to reuse code across projects."

"quirks" seems like a bit close to "bugs".

The way I read it sounds like the window backends as oppositions in some way. They are just implementations that goes as far as the features supported by the underlying API. We try to make them as consistent as possible, and it isn't usually noticeable from a user perspective which window backend you use. You ship applications with whatever API that works for the users. Extra features are only in backends that use customized events, which are intended for stuff like theme park simulators, IoT and such that uses custom hardware. The window backends for GLFW/SDL2/Glutin are very similar in behavior. Would like to avoid confusion around this point.